src/index.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html>
+    <body>
+        <div id="root"></div>
+        <script src="./index.js"></script>
+    </body>
+</html>
```

Comment:
Webpack uses this file as a template when bundling everything together. I'm just telling it to have an div with an id of root so that our React app knows where to attach itself. 

Basically, anything in this file will be there before the React code finishes running.